pinot-server/src/test/java/org/apache/pinot/server/api/BaseResourceTest.java

```diff
@@ -55,11 +56,12 @@
 public abstract class BaseResourceTest {
   private static final String AVRO_DATA_PATH = "data/test_data-mv.avro";
   private static final File INDEX_DIR = new File(FileUtils.getTempDirectory(), "BaseResourceTest");
-  protected static final String TABLE_NAME = "testTable";
+  protected static final String REALTIME_TABLE_NAME = "testTable_REALTIME";
```

Comment:
nit: Use the TableNameBuilder. let TABLE_NAME still be there, and you can use the builder to derive the other two

Reply:
+1

Reply:
DONE.